be/src/runtime/exec_env.cpp

```diff
@@ -233,6 +236,25 @@ const std::string& ExecEnv::token() const {
     return _master_info->token;
 }
 
+void ExecEnv::add_rf_event(const TUniqueId& query_id, int filter_id, const std::string& network, std::string&& msg) {
+    _runtime_filter_cache->add_rf_event(query_id, filter_id, strings::Substitute("$0($1)", msg, network));
```

Comment:
`msg` is a rvalue reference, but still a lvalue. Maybe we should use `strings::Substitute("$0($1)", std::move(msg), network)` to avoid copying it.